auth-api/src/auth_api/services/membership.py

```diff
@@ -143,8 +156,14 @@ def update_membership(self, updated_fields, token_info: Dict = None):
     def deactivate_membership(self, token_info: Dict = None):
         """Mark this membership as inactive."""
         current_app.logger.debug('<deactivate_membership')
+        # if this is a member removing another member, check that they admin or owner
         if self._model.user.username != token_info.get('username'):
             check_auth(token_info=token_info, one_of_roles=(ADMIN, OWNER))
```

Comment:
I know this is old code. Just wondering whether this would work as there are no org_id or business_identifier passed as args.

Reply:
Good catch, I've added the `org_id` param